pkg/apis/core/v1beta1/defaults.go

```diff
@@ -52,7 +52,13 @@ func SetDefaults_Seed(obj *Seed) {
 	}
 
 	if obj.Spec.Settings.ExcessCapacityReservation == nil {
-		obj.Spec.Settings.ExcessCapacityReservation = &SeedSettingExcessCapacityReservation{Enabled: true}
+		obj.Spec.Settings.ExcessCapacityReservation = &SeedSettingExcessCapacityReservation{
+			Enabled: false,
+			Configs: []SeedSettingExcessCapacityReservationConfig{
+				// This roughly corresponds to a single, moderately large control-plane.
+				{Resources: corev1.ResourceList{corev1.ResourceCPU: resource.MustParse("2"), corev1.ResourceMemory: resource.MustParse("6Gi")}},
+			},
+		}
```

Comment:
We should also add the default config to the configs when `ExcessCapacityReservation` is non-nil but `Configs` is empty.

Reply:
Wouldn't this enable the function also when you provide explicitly an empty `Config` in order to disable this functionality?

Reply:
Ah, That's true.

Reply:
> Wouldn't this enable the function also when you provide explicitly an empty Config in order to disable this functionality?

TBH, it's kind of odd to disable `ExcessCapacityReservation` by configuring an empty object. I also wonder if that even works because we have `omitEmpty` in place?!